src/Symfony/Component/RateLimiter/Policy/SlidingWindow.php

```diff
@@ -114,6 +114,18 @@ public function __serialize(): array
 
     public function __unserialize(array $data): void
     {
+        // BC layer for old objects serialized via __sleep
+        if (\count($data) >= 5) {
```

Comment:
Can we be more specific? 
```suggestion
        if (\count($data) === 5) {
```

Reply:
I guess it doesn't hurt 🤷🏻‍♂️